crud/get.lua

```diff
@@ -77,7 +137,9 @@ local function call_get_on_router(space_name, key, opts)
         tuple = nil
     end
 
-    return utils.format_result({tuple}, space)
+    local formatted_result = utils.format_result({tuple}, space)
```

Comment:
What if `utils.format_result` accept `fields` argument?

`return utils.format_result({tuple}, space, fields)`

In this case you can just change this function call everywhere you need.